package.json

```
@@ -3,6 +3,7 @@
   "scripts": {
     "dev": "yarn run development",
     "development": "cross-env NODE_ENV=development webpack --progress --hide-modules --config=webpack.config.js",
+    "eslint": "eslint 'resources/assets/lib/**/*.{js,jsx,ts,tsx}' *.js",
```

To include `.eslintrc.js`

```suggestion
    "eslint": "eslint 'resources/assets/lib/**/*.{js,jsx,ts,tsx}' '*.js'",
```